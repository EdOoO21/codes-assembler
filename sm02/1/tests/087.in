511 -613 792 718 665 107 -791 -425 544 -314 320 112 444 -329 803 453 859 -750
