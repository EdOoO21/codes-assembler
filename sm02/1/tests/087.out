-750
859
453
803
-329
444
112
320
-314
544
-425
-791
107
665
718
792
-613
511
